src/zc/Zc.adoc

```diff
@@ -0,0 +1,396 @@
+//:sectnums:
+//:version-label: v1.0.4-2
+//:lifecycle-state: ratified
+
+[#Zc]
+== Zc* {version-label}
+
+=== Change history since v0.70.1 (tagged release)
+
+.Change history
```

Comment:
do we need the revision history in the unpriv spec? other extensions don't have that